backend/src/interface/persistent_data/mod.rs

```diff
@@ -1 +1,141 @@
 //! The interfaces specified here allow access to data stored in a persistent datastore like a database.
+use crate::interface::persistent_data::model::{
+    ApiKey, Canteen, DataError, Image, ImageInfo, Line, Meal, Side,
+};
```

Comment:
maybe its prettyer to import `model::*`?
Depends on what the formater would do with that...

Reply:
Idk if this is really important. But if clippy wants to, we can change it.